### PR TITLE
telco5g: Add a few new ginkgov2 tweaks

### PR DESCRIPTION
**Pull request by sshnaidm**
[CNF-8880](https://issues.redhat.com//browse/CNF-8880)

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -203,13 +203,31 @@ function sno_fixes {
     popd
 }
 
+function get_time_left {
+
+    now=$(date +%s)
+    then=$(cat $SHARED_DIR/start_time)
+    minutes_passed=$(( (now - then) / 60 ))
+    time_left=$(( 225 - minutes_passed ))  # the job has 4 hours to run, leave 15 minutes for must-gathering
```
Comment (yuvalk) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:211]: if properly using post- steps
then must-gather is accounted in the extra 1-hour you've got for tear-down
https://docs.ci.openshift.org/docs/architecture/timeouts/#ci-operator-test-process-timeouts

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:211]: Yeah, seems like it's useful for `configsuite` that runs before e2e tests and takes about 20 minutes

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 2]
```diff
@@ -267,24 +285,24 @@ popd
 # Skiplist common for all releases
 create_tests_skip_list_file
 
-# Skiplist according to each release
+# Skiplist according to each release and add flakey parameter for Ginkgo v1 and v2
 if [[ "$CNF_BRANCH" == *"4.11"* ]]; then
     create_tests_temp_skip_list_11
-    export GINKGO_PARAMS='-ginkgo.slowSpecThreshold=0.001 -ginkgo.v -ginkgo.progress -ginkgo.reportPassed'
+    export GINKGO_PARAMS="-ginkgo.slowSpecThreshold=0.001 -ginkgo.v -ginkgo.progress -ginkgo.reportPassed -ginkgo.flakeAttempts 4"
```
Comment (yuvalk) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:291]: so all our tests are flaky?

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:291]: Not all of them of course.

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-sno-cnftests periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Pull request merged**